frontend/src/pages/EmptyCategoryPage.tsx

```diff
@@ -0,0 +1,83 @@
+import { Stack, Typography, Button } from '@mui/material';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import AddIcon from '@mui/icons-material/Add';
+import { StringBank } from '../strings';
+import { useTranslation } from 'react-i18next';
+
+interface State {
+  catName: string;
+  catColor: string;
+}
+
+const Span = styled.span`
+  && {
+    background-color: ${(props) => props.color};
+    width: 54px;
+    height: 25px;
+    flex-grow: 0;
+    padding: 2px 6px 3px;
+    border-radius: 4px;
+    font-size: 14px;
+    margin: 0 6px;
+    font-weight: normal;
+    letter-spacing: normal;
+  }
+`;
+
+const NewAgreementButton = styled(Button)`
+  && {
+    width: 160px;
+    height: 36px;
+    border-radius: 8px;
+    flex-grow: 0;
+    font-family: Lato;
+    font-size: 14px;
+    background-color: #3f4550;
+  }
+`;
+
```

Comment:
Let's refactor this - minimum styling from here, maximum from theme. No need for button color because it uses that color by default (configured in theme)